core/src/main/java/cz/o2/proxima/repository/AttributeFamilyDescriptor.java

```diff
@@ -135,6 +89,45 @@ public static Builder newBuilder() {
     this.access = Objects.requireNonNull(access);
     this.filter = filter;
     this.source = source;
+
+    this.replicationConsumerNameFactory =
```

Comment:
We can delegate all this to call to `setup`. That way, we can easily introduce a more complex logic (replace the ConsumerNameFactory with completely custom logic) in future.

Reply:
changed.